485 display middle name

rmathew1011 opened the pull request:
# Description

Please include a summary of the changes. Please also include relevant motivation and context. At a minimum, providing a bulleted list of changes in your PR is helpful to reviewers.

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# Testing Procedures

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
  - After updating the middle name predicate in the SPARQL file, the display template was modified to render the middleName property.

 Steps to verify the changes:

- On the local Solr UI, ran a query to confirm that the middleName property was being indexed and rendered correctly.

- Queried the list of scholars who have a middle name to ensure the property appears consistently.

- Accessed the Scholars application in a browser and verified that the middle name is displayed properly for these individuals.

tamu-sad-iii commented on src/main/resources/defaults/displayViews/persons/mainContentTemplate.html:
```diff
@@ -1,5 +1,5 @@
 <div>
-  <span class="h3">{{name}}</span>
+  <span class="h3"> {{lastName}}, {{firstName}} {{middleName}}</span>
```
There is an extra space in the beginning. Additionally, what if `firstName` and `middleName` are not present. I recommend building this conditionally. It gets messy in handlebars and could be a candidate for a handlebar helper.

rmathew1011 replied:
Removed extra space before lastName
Utilized inline handlebars to conditionally render firstName and middleName.

qtamu reviewed (commented):
This pr is pulling in changes not associated with the issue.

rmathew1011 committed:
485 - to display the middle name.

 - Removed extra space from the beginning.
 - Used inline handlebar to conditionally render the irst name and the middle name.

rmathew1011 commented:
> This pr is pulling in changes not associated with the issue.

Kevin helped me fixed this issue - rebasing fixed this issue.

rmathew1011 committed:
WIP - adding the handlebar helper for name display

rmathew1011 committed:
Added allNamesPresent handler bar to conditionally render name field

rmathew1011 committed:
Update mainContentTemplate.html

The changes to the span class styles were unintentional and only made for testing purposes. I’ve reverted them back to 
`<span class="h3"></span>`
since the style updates were not requested.

tamu-sad-iii commented:
Please link to the scholars-embed-utilities PR for the handlebar helper and the PR upgrading scholars-angular and scholars-embed-utilities to use the next release of scholars-embed-utilities.
